ansibullbot/utils/file_tools.py

```diff
@@ -44,6 +44,7 @@ def __init__(self, botmetafile=None, checkoutdir=None, repo=None):
         self.FILEMAP = {}
         self.match_cache = {}
         self.update(force=True)
+        self.email_commits = {}
```

Comment:
`email_commits` should be initialized here (or a property should be used for `email_commits`).